<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Rizwan-PC" name="Command line test 0741f328-0c19-4193-8e35-2148a343a379" tests="3" failures="3" timestamp="26 Apr 2021 05:54:44 GMT" time="53.163" errors="0">
  <testcase name="login" time="16.351" classname="org.testing.TestScripts.TC1">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 1, Size: 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 1, Size: 0
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at org.testing.TestScripts.TC1.login(TC1.java:40)
	at org.testing.runner.Trigger.main(Trigger.java:72)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase name="login" time="0.606" classname="org.testing.TestScripts.TC3">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 1, Size: 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 1, Size: 0
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at org.testing.TestScripts.TC3.login(TC3.java:26)
	at org.testing.runner.Trigger.main(Trigger.java:72)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase name="login" time="0.196" classname="org.testing.TestScripts.TC4">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 1, Size: 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 1, Size: 0
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at org.testing.TestScripts.TC4.login(TC4.java:27)
	at org.testing.runner.Trigger.main(Trigger.java:72)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- login -->
</testsuite> <!-- Command line test 0741f328-0c19-4193-8e35-2148a343a379 -->
